### PR TITLE
dev-cpp/valijson: bump to 1.0

### PR DESCRIPTION
**Pull request by hhoffstaette**
This also removes the test support since upstream has removed the vendored test dependencies from the tarball in favor of git submodules.

Closes: https://bugs.gentoo.org/891683
Signed-off-by: Holger Hoffstätte <holger@applied-asynchrony.com>

### CODE REVIEW

**Comment by hhoffstaette** [comment]
I removed the Closes: tag in order to keep the bug open until stabilization.

**Comment by hhoffstaette** [comment]
@thesamesam Can I have a bump? The version change is mostly formal and has no compatibility implications, just getting in line with some fixes & comments from upstream.
The loss of tests is IMHO silly but nothing I can do about right now.

File: dev-cpp/valijson/valijson-1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+inherit cmake
+
+DESCRIPTION="Header-only C++ library for JSON Schema validation"
+HOMEPAGE="https://github.com/tristanpenman/valijson"
+SRC_URI="https://github.com/tristanpenman/valijson/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE=""
```
Comment (ceamac) [inline dev-cpp/valijson/valijson-1.0.ebuild:15]: Please remove empty `IUSE`

File: dev-cpp/valijson/files/1.0-fix-macro-typos.patch [hunk 2]
```diff
@@ -0,0 +1,73 @@
+From f1ff3518281fe347c97b78384d6bb645dbdd4788 Mon Sep 17 00:00:00 2001
```
Comment (ceamac) [inline dev-cpp/valijson/files/1.0-fix-macro-typos.patch:1]: Can you also add the link where this patch originates from?  It is a github commit, but a link is better.

File: dev-cpp/valijson/valijson-1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Header-only C++ library for JSON Schema validation"
+HOMEPAGE="https://github.com/tristanpenman/valijson"
+SRC_URI="https://github.com/tristanpenman/valijson/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
```
Comment (ceamac) [inline dev-cpp/valijson/valijson-1.0.ebuild:12]: There is one file licensed under `Boost-1.0`, please update

**Review by ceamac — CHANGES_REQUESTED** [review]
A few small changes, looks good otherwise.

Thank you!

**Comment by hhoffstaette** [comment]
Thank you for having a look, I had almost forgotten about it. :sweat: 
All the mentioned issues are resolved.

**Review by ceamac — APPROVED** [review]

**Comment by ceamac** [comment]
I added a Bug: tag instead of the Closes: you removed, so the commit can be seen in the bug.

**Pull request closed without merging**